# Define the goal
#int32 chapa_id
int32 rectangle_id #0,90,180,270

int32 operation_type

geometry_msgs/Polygon rectangle


---
# Define the result
bool recorrido
geometry_msgs/Point[] path
---
# Define a feedback message
